casper/src/test/scala/coop/rchain/casper/sync/BlockRetrieverRequesAllSpec.scala

```diff
@@ -31,7 +31,7 @@ import monix.execution.Scheduler.Implicits.global
 
 import scala.concurrent.duration._
 
-class BlockRetrieverRequesAllSpec extends FunSpec with BeforeAndAfterEach with Matchers {
+class BlockRetrieverRequestAllSpec extends FunSpec with BeforeAndAfterEach with Matchers {
```

Comment:
This test have some logical flaw. E.g. if only one peer is left in waiting list, block has to be requested from this peer, but test expects that block is not requested and broadcast of HasBlockRequest happening instead.

Reply:
What are you suggesting as a solution?

Reply:
The changes are the solution. I meant that changes in this test are justified by the fact that previously test had logical flaws that are fixed.